package.json

```diff
@@ -32,9 +32,9 @@
   },
   "dependencies": {
     "@backstage/catalog-model": "^1.4.1",
-    "@backstage/core-components": "^0.13.3",
+    "@backstage/core-components": "^0.13.4",
     "@backstage/core-plugin-api": "^1.5.3",
-    "@backstage/plugin-catalog-react": "^1.8.0",
+    "@backstage/plugin-catalog-react": "^1.8.3",
```

Comment:
Upgrade helper says to upgrade to 1.8.1, is it wrong?  https://backstage.github.io/upgrade-helper/?from=1.16.0&to=1.17.4

Reply:
I think it's just that the upgrade-helper is not really updating that in the patched versions. The version has been automatically picked up by the versions:bump command.